ci/scripts/evaluate_commits.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Copyright IBM Corp. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+set -euo pipefail
+
+if [[ $(git diff-tree --no-commit-id --name-only 'HEAD^..HEAD') == "docs" ]]; then
```

Comment:
would `HEAD^..HEAD` work as expected if a PR contains multiple commits?

Reply:
I tested this with the following commit chains:
- first commit touches doc, second touches code, ran against CI, and it triggered all jobs as expected
- first touches doc, second touches code, third touches doc, all jobs ran as expected
- Third change I ran with a single commit to code, all jobs ran as expected.
- Fourth change I ran with just a doc change, only the docbuild ran.

Reply:
Thanks for bringing that up, hadn't run it against the second case, but I've confirmed it works as expected now